### PR TITLE
🌐 Add Tamil translation for `docs/ta/docs/features.md`

### PR DESCRIPTION
**Pull request by arunppsg**
#5434 

This PR adds translation of `features.md` to tamil.

### CODE REVIEW

**Comment by tiangolo** [comment]
Great! Let me know when the PR is ready for review and I'll add the label `awaiting review` so that others can come and review it and then I can merge it. :rocket:

**Comment by tiangolo** [comment]
📝 Docs preview for commit f15f6bc0ea47fe57de3796b69eb9dc22a710a40e at: https://649a2f0297baca1ed99aa116--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit 559aafe2f3b4d1654d55376b5b38ce967d09effb at: https://649a31ce1636a4208c534ca6--fastapi.netlify.app

**Comment by tiangolo** [comment]
Great @dineshsrd! Yes, please help review the PR.

You can see the instructions here: https://fastapi.tiangolo.com/contributing/#translations that includes a link with instructions to add a review to a PR.

**Comment by tiangolo** [comment]
@dineshsrd, it would be great if you could add your review to the PR. You would have to go to the "Files" tab and there click the button to add your review.

**Comment by codespearhead** [comment]
@dineshsrd If you happen to know two other Tamil-speaking developers that you can also review this PR, even better. [Here's how you can review PRs btw](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review).

File: docs/ta/docs/features.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+# A Pydantic model
+class User(BaseModel):
+    id: int
+    name: str
+    joined: date
+```
+
+That can then be used like:
```
Comment (dineshsrd) [inline docs/ta/docs/features.md:52]: Please add translation here.

File: docs/ta/docs/features.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+* in <a href="https://code.visualstudio.com/" class="external-link" target="_blank">Visual Studio Code</a>:
+
+![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+* in <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a>:
+
+![editor support](https://fastapi.tiangolo.com/img/pycharm-completion.png)
+
+நிங்கள்
```
Comment (dineshsrd) [inline docs/ta/docs/features.md:89]: Please complete the translation for rest of the docs.

**Review by dineshsrd — CHANGES_REQUESTED** [review]
Please do complete the translation for rest of the doc.

**Comment by dineshsrd** [comment]
@tiangolo Review done and comments added.

**Comment by arunppsg** [comment]
@dineshsrd thank you for the review. i will update the docs with suggestions by end of this week.

**Comment by arunppsg** [comment]
I am closing this pull request as I am not able to prioritize time for this, sorry!

**Pull request closed without merging**